apps/explorer/.env

```diff
@@ -2,7 +2,7 @@ NX_CHAIN_EXPLORER_URL=https://explorer.vega.trading/.netlify/functions/chain-exp
 NX_TENDERMINT_URL=https://n01.stagnet3.vega.xyz/tm
 NX_TENDERMINT_WEBSOCKET_URL=wss://n01.stagnet3.vega.xyz/tm/websocket
 NX_VEGA_CONFIG_URL=https://static.vega.xyz/assets/stagnet3-network.json
-NX_VEGA_NETWORKS='{"TESTNET":"https://explorer.fairground.wtf","MAINNET":"https://explorer.vega.xyz"}'
+NX_VEGA_NETWORKS={\"MAINNET"\:\"https://explorer.vega.xyz"\,\"TESTNET\":\"https://explorer.fairground.wtf\",\"DEVNET"\:\"https://dev.explorer.vega.xyz"\,\"STAGNET1\":\"https://stagnet1.explorer.vega.xyz\",\"STAGNET3\":\"https://stagnet3.explorer.vega.xyz\"}
```

Comment:
We don't publisice our internal test networks. This should only be testnet and mainnet